influence_benchmark/experiments/KTO_experiment.py

```diff
@@ -8,16 +8,18 @@
 
 
 def main():
+    testing = True
     env_name = "therapist"
-    max_turns = 5
-    num_envs_per_device = 12
+    max_turns = 5 if not testing else 2
+    num_envs_per_device = 12 if not testing else 2
```

Comment:
we likely want num_envs_per_device = 12 (for A100) or 8 (for A6000) for testing as well, lowering this number will slow things down, not speed things up?

Reply:
Yeah, I commented about this above in the main PR description